.github/workflows/tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Tests
+
+on:
+    push:
+    pull_request:
+
+jobs:
+
+    redis-tests:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: [7.2, 7.3, 7.4]
```

Comment:
We need to consider if it's worth to run this kind of integration tests on all supported PHP versions or not.